README.md

```diff
@@ -16,6 +27,32 @@ Golang implementation of Celestia's data availability node types (`light` | `ful
 ### Installation
 
 ```sh
-git clone https://github.com/celestiaorg/celestia-node.git
+git clone https://github.com/celestiaorg/celestia-node.git 
+```
+```
 make install
 ```
+
+### Node types
+* **Bridge** nodes - bridge nodes relay blocks from the celestia consensus network to the celestia data availability 
+(DA) network
+* **Full** nodes - full nodes fully reconstruct and store blocks by sampling the DA network for shares
+* **Light** nodes - light nodes verify the availability of block data by sampling the DA network for shares
```

Comment:
👍🏻 

Not sure why do you duplicate "<type> nodes - <type> nodes", e.g. **Bridge** nodes - bridge nodes

Reply:
good point, resolved?